firmware/badge/apps/nametag.py

```diff
@@ -244,7 +244,7 @@ def run_foreground(self):
                 # Update menubar for picker controls: F1 Select, F3 Prev, F4 Next, F5 Cancel
                 try:
                     self.p.set_menubar_button_label(0, "Select")
-                    self.p.set_menubar_button_label(1, "")
+                    self.p.set_menubar_button_label(1, "Hide Img")
```

Comment:
I would like to see this on line 393 if possible.  That's in the main menu for the nametag.